static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -524,7 +524,7 @@ const DefaultLine = styled('div')<{
   justify-content: space-between;
   align-items: center;
   background: ${p => (p.isSubFrame ? `${p.theme.surface100}` : `${p.theme.surface200}`)};
-  min-height: 32px;
```

Comment:
<img width="400" alt="image" src="https://github.com/user-attachments/assets/fdb12e35-d1ba-4e11-babf-b4f4491c6004">

Reply:
nevermind doesn't seem like this can be done consistently.

Reply:
Changed to `size='zero'` with some font size changes